### ISSUE TITLE
Add an operation mode of SMS only

### ISSUE THREAD

**Issue by jd1378** [jd1378/otphelper#86]
Currently app relies on reading notifications for service, this is buggy on different android versions
as an alternative for users, it's a good idea to add an SMS-only mode of operation

**Comment by jd1378** [comment]
not going to implement

**Comment by pataquets** [comment]
@jd1378 Just out of curiosity... what is the reason?
Thinking about possible causes, only security (requesting SMS permission) comes to mind.
However, as (possible) advantages, less events to filter thru for sure and... maybe Android has a specific event for waking up the app on SMS arrival, which I guess would improve battery usage. Warning: no Android dev here, just guessing.

**Comment by jd1378** [comment]
Well I just think the current mode of operation is working fine
the idea was just what I came up with after it was not working on android 10 as expected, which in 1.16.0-rc it seems to have been resolved (not confirmed myself yet)

I haven't seen the usage battery even showing up yet. which usually means the battery usage is minimal. let me know if that's not the case.

did you have any problem with app going to sleep so far ?
